tests/unit/worker/test_submission_worker.py

```diff
@@ -43,7 +123,27 @@ def test_create_dir_as_python_package(self):
 
     def test_return_file_url_per_environment(self):
         returned_url = return_file_url_per_environment(self.url)
-        self.assertEqual(returned_url, "http://testserver/test/url")
+        self.assertEqual(returned_url, "{0}{1}".format(self.testserver, self.url))
+
+    @mock.patch("scripts.workers.submission_worker.SUBMISSION_DATA_DIR", "/tmp/test-dir/compute/submission_files/submission_{submission_id}")
+    @mock.patch("scripts.workers.submission_worker.SUBMISSION_INPUT_FILE_PATH", "/tmp/test-dir/compute/submission_files/submission_{submission_id}/{input_file}")
+    @mock.patch("scripts.workers.submission_worker.create_dir_as_python_package")
+    @mock.patch("scripts.workers.submission_worker.download_and_extract_file")
+    def test_extract_submission_data_successful(self, mock_download_and_extract_file, mock_create_dir_as_python_package):
+        submission = extract_submission_data(self.submission.pk)
+        expected_submission_data_dir = "/tmp/test-dir/compute/submission_files/submission_{submission_id}".format(submission_id=self.submission.pk)
```

Comment:
submission path prefix is being used at multiple places can we create a class variable for this `tmp/test-dir/compute/submission_files`

Reply:
Then is it okay to add class variables for all paths [as in `submission_worker.py`](https://github.com/Cloud-CV/EvalAI/blob/master/scripts/workers/submission_worker.py#L56-L66)? It will help for creating other tests like that for `extract_challenge_data`.